spark/src/test/scala/org/apache/spark/sql/CometTestBase.scala

```diff
@@ -259,7 +259,7 @@ abstract class CometTestBase
       dfSpark.queryExecution.explainString(ExtendedMode),
       dfComet.queryExecution.explainString(ExtendedMode))
     if (supportsExtendedExplainInfo(dfSpark.queryExecution)) {
-      assert(diff.contains(expectedInfo))
+      assert(expectedInfo.exists(s => diff.contains(s)))
```

Comment:
Don't we need to have all expected info in the diff?

Reply:
Thanks

Updated